.github/workflows/main.yml

```diff
@@ -4,7 +4,16 @@ jobs:
   publish:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v3
+    - uses: actions/checkout@v4
+    - uses: actions/checkout@v4
+      with:
```

Comment:
Why the same action twice, first one without args?

Reply:
- The first one (wihout args / with all defaults) checks out this repo (quarkus-mpxj).
- The second checks out a part of Yolean/yolean.github.io like this https://github.com/actions/checkout#checkout-multiple-repos-nested

Reply:
I forgot that git clone isn't implicit 🙂